website/content/partials/vagrant-cloud-deprecation.mdx

```diff
@@ -0,0 +1,5 @@
+<Note>
+
+HashiCorp has deprecated Vagrant Cloud and migrated the boxes to HCP Vagrant Registry. Read the [documentation](https://developer.hashicorp.com/hcp/docs/vagrant) to learn more about HCP Vagrant Registry.
```

Comment:
```suggestion
HashiCorp has moved Vagrant Cloud and migrated the boxes to HCP Vagrant Registry. Read the [documentation](https://developer.hashicorp.com/hcp/docs/vagrant) to learn more about HCP Vagrant Registry.
```

We're ensuring that all previous domains are routed to HCP so nothing is broken or lost. Also deprecation makes it sound like we don't want them to use the service anymore, which isn't true.